plugins/woocommerce-admin/client/wp-admin-scripts/print-shipping-label-banner/shipping-banner/index.js

```diff
@@ -390,10 +362,29 @@ export class ShippingBanner extends Component {
 			showShippingBanner,
 			isShippingLabelButtonBusy,
 		} = this.state;
+		const { isWcstCompatible } = this.props;
+		if ( ! showShippingBanner && ! isWcstCompatible ) {
+			document
+				.getElementById( 'woocommerce-admin-print-label' )
+				.classList.add( 'error' );
+
+			return (
+				<p>
+					<strong>
+						{ __(
+							'Please update the WooCommerce Shipping & Tax plugin to the latest version to ensure compatibility with WooCommerce Shipping.',
```

Comment:
I don't think we have a function to update the plugin - although that would be sweet to offer here 😄 

Would a link to the plugins page maybe make sense as a way to guide merchants to a next step?

Reply:
Addressed in https://github.com/woocommerce/woocommerce/pull/50970/commits/8c49a3e049db82a3c276638b0dd0fc30facec264